From c9b85043f76e48fc6f74b22522f438d5530763d4 Mon Sep 17 00:00:00 2001
From: Andy Green <andy@openmoko.com>
Date: Fri, 25 Jul 2008 23:06:10 +0100
Subject: [PATCH] fix-remove-unused-model-name-bq27000.patch

Model name isn't in the bq27000 register set, remove the
claim that we can deliver it

Signed-off-by: Andy Green <andy@openmoko.com>
---
 drivers/power/bq27000_battery.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/power/bq27000_battery.c b/drivers/power/bq27000_battery.c
index d19186a..aee8018 100644
--- a/drivers/power/bq27000_battery.c
+++ b/drivers/power/bq27000_battery.c
@@ -269,7 +269,6 @@ static enum power_supply_property bq27000_battery_props[] = {
 	POWER_SUPPLY_PROP_PRESENT,
 	POWER_SUPPLY_PROP_TIME_TO_EMPTY_NOW,
 	POWER_SUPPLY_PROP_TIME_TO_FULL_NOW,
-	POWER_SUPPLY_PROP_MODEL_NAME,
 	POWER_SUPPLY_PROP_CAPACITY
 };
 
-- 
1.5.6.3

